.github/workflows/publish.yml

```diff
@@ -19,14 +19,30 @@ jobs:
 
     steps:
     - uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
+
     - name: Set up Python
       uses: actions/setup-python@v4
       with:
         python-version: "3.11"
+        
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
         pip install build
+
+    - name: Check git status
+      run: |
+        git status
+        git log -n 2
+        git diff
+
+    - name: Install WallGo Python package
+      run: |
+        pip uninstall WallGo
+        pip install . --upgrade -v
```

Comment:
Is the uninstall necessary? I'm under the impression that github runners always start with a fresh virtual environment

Reply:
Yes, it is. Again, just for debugging purposes. I could remove it another day, but for now I think let's proceed, as we approach the submission deadline.